packages/nextjs-kit/src/components/footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Link from 'next/link';
+import { Menu } from '../index';
+
+interface MenuProps {
+  menuItems: Menu[];
+  children: JSX.Element;
+}
+
+/**
+ * This is a Footer component.
+ *
+ * @param {Props} menuItems holds the title, href, and id of each route to be displayed
+ * @param {Props} children  copyright display and information to be passed in by the user
+ * @returns {React.JSX.Element}
+ */
+const Footer: React.FC<MenuProps> = ({ menuItems, children }: MenuProps) => {
+  const FooterMenu = () => (
+    <nav className="flex flex-col max-w-lg mx-auto lg:max-w-screen-lg">
+      <ul>
+        {menuItems.map(({ label, path, id }) => {
```

Comment:
When I tried to use this component by passing in our "footerMenu" coming back from Drupal, I don't see any of the content: 
<img width="603" alt="image" src="https://user-images.githubusercontent.com/63015754/185153899-5a20f66b-56c2-43ba-b2ff-39151aecdfa6.png">

Also if possible, retain the logic that makes a menu item that has a parent indented like so:

- parent item 1
  - child item 1
  - child item 2
- parent item 2

Reply:
Sorry I should have left a comment about this. Our FooterMenu items in the WP starter are named differently than in the Drupal starter so changes need to be made to the object passed in one of the starters for it work with both. I imagined this work would take place in another ticket as it is not related to the components here but rather how the starter kits build these objects.

Naming is `label, path, id` vs `title, url, id`

Reply:
Ah I see. In that case these should be a bit more generic. I guess I would expect equal work to implement this component in `next-drupal` and `next-wordpress`.

There's an opportunity here to combine these two interfaces here as well. For example:

```
export interface LinkProps {
  linkText: string;
  href: string;
}

export interface FooterMenuItem extends LinkProps {
  id: string;
}
```

This way, the header component's prop types can refer to the more generic `LinkProps` interface, which the footer menu is essentially an extension of, then we can have the same props `href` and `linkText` (maybe a better name for this) in both components, giving us some cohesion.